src/models/lib/cache.js

```diff
@@ -0,0 +1,117 @@
+/*****
```

Comment:
If this cache is specific to the participantEndpoints, then it should go under the domain folder (with file rename) as follows `src/domain/participant/lib/cache/ParticipantEndpoints.js` (once you rename the folder).

As per our conversation, if there is anything *generic*, then it should be moved to `src/lib/cache` folder. This can be done in future once we have some more caching requirements.